### PR TITLE
fix:Google oauth

### PR DESCRIPTION
**Pull request by Pranay105**
- Updated the content as per the feedback

### CODE REVIEW

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:40]: Can we simplify this note? 
```suggestion
* Test the Client ID and Client Secret before disabling Form login on Appsmith. This ensures that users can access Appsmith with their Google credentials. To fix an invalid Google configuration, update the environment variables.
```

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:41]: These two pieces of information do not tie up together. Maybe a rewrite could be:
```suggestion
* To fix signup issues, if you are running appsmith behind an ELB / Proxy, see [OAuth Sign up not working](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```
P.S. - can we get this confirmed by the technical team?

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:54]: Let's add a short intro for Configuring Appsmith Instance, which tells users that there are below ways in which the instance could be configured:
* Admin Settings(Add links to its subsection)
* Environment Variables(Add links to its subsection)

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:43]:
```suggestion
Navigate to the `docker.env` file located in `<PROJECT_ROOT>/stacks/configuration/`. Update the values for the following keys:
```

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:57]: Create a subhead for Admin Settings.

File: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md
Comment (jnikhila) [inline website/docs/getting-started/setup/instance-configuration/authentication/google-login.md:43]: Create a subhead for Environment Variables.

**Review by jnikhila — CHANGES_REQUESTED** [review]

**Comment by infinitetrooper** [comment]
@jnikhila @Pranay105  - We're adding the Javascript Origin URI and Redirect URI in the Google OAuth config screen on Appsmith so the copy-paste experience is smoother (Image attached below), this will go in the next release so, can we wait for a little before we merge this and include the updated screenshot and process also?

Screenshot 2023-03-10 at 5 22 31 PM

**Comment by jnikhila** [comment]
> @jnikhila @Pranay105 - We're adding the Javascript Origin URI and Redirect URI in the Google OAuth config screen on Appsmith so the copy-paste experience is smoother (Image attached below - there's a typo there), this will go in the next release so, can we wait for a little before we merge this and include the updated screenshot and process also?
> 
> [appsmithorg/appsmith#21170](https://github.com/appsmithorg/appsmith/issues/21170)
> 
> Screenshot 2023-03-10 at 5 22 31 PM

@infinitetrooper - let's add a new ticket for this change, and we can merge the changes along with #[21170](https://github.com/appsmithorg/appsmith/issues/21170)

**Pull request merged**
